ark/utils/load_utils.py

```diff
@@ -130,16 +130,16 @@ def load_imgs_from_tree(data_dir, img_sub_folder=None, fovs=None, channels=None,
         # need this to reorder channels back because list_files may mess up the ordering
         channels_no_delim = [img.split('.')[0] for img in channels]
 
-        channels = iou.list_files(
+        all_channels = iou.list_files(
```

Comment:
Do we need to modify load_imgs_from_dir as well?

Reply:
I don't think so. In `load_imgs_from_dir`, all we use `list_files` for is to get the images we need to load in the directory. We don't need to read in specific channel files because the channels exist in a dimension in each image to load. Whereas in `load_imgs_from_tree`, the channel data is contained in separate files which we have to read in manually.

Reply:
Should we set exact_match=True for load_imgs_from_dir?

Reply:
I think you could make a reasonable argument for either case

Reply:
@ngreenwald I think we should just use `exact_match=False` since we're only doing file extension substr matching from what I can see.